WCAG 2.2 guidance updates (#3090)
alphagov/govuk-design-system: src/patterns/contact-a-department-or-service-team/index.md

---

{%- from "_example.njk" import example -%}
{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
{% from "govuk/components/tag/macro.njk" import govukTag %}

Give users contact information within your service.

{% set wcagCallout %}

{{ govukTag({
  text: "WCAG 2.2",
  classes: "app-tag"
}) }}

### New WCAG 2.2 criteria affects this pattern

To help users to 'Contact a department or service team' and meet the new WCAG 2.2 criteria, make sure that users can successfully:

> - [find contact details to get help in a consistent place on each page](/patterns/contact-a-department-or-service-team/#wcag-consistent-contact)

Author: ```suggestion
- [find help details in a consistent place across contact pages](/patterns/contact-a-department-or-service-team/#wcag-consistent-contact)
```

Reviewer: Same here, this doesn't clarify that it's for help with the same content. Although these links don't need to be that detailed, so don't necessarily need to change.

{{ example({ group: "patterns", item: "contact-a-department-or-service-team", example: "all-contact-information", html: true, open: false, size: "xl" }) }}

Order contact channels based on what research shows your users need, and what your service or department can best support.

Show contact channels in the same order throughout your service. This helps users to find what they need more easily.

<div class="app-wcag-22" id="wcag-consistent-contact" role="note">
  {{ govukTag({
    text: "WCAG 2.2",
    classes: "app-tag"
  }) }}

>   <p>If you use this pattern across a set of pages, make sure to place the contact information consistently on each page.</p>
>   <p>For example, if you place a ‘contact us’ section at the end of one page within a set of pages, then you should place any ‘contact us’ sections on other pages within the set at the end of the other pages as well.</p>

Author: ```suggestion
  <p>If you use this pattern across a set of pages, make sure to place the contact information consistently on each page.</p>
  <p>For example, if you place a 'contact us' section at the end of one help page, any other help pages that include a 'contact us' section should have them placed at the end as well.</p>
```

Reviewer: The change seems to say the pages that have the pattern need to have the same content, while its the content of the pattern, not the rest of the pages.

For example, a department might have three sets of different contact details, one for general enquiries, one for complaints and one for reporting fraud. Those details will have different content. If you use details for general enquiries on multiple pages, then they should be in the same relative order. But the details for complaints could be in a different order than the details for general enquiries. The details only need to be in the same relative order as details of the same content.

Author: Edit Anika on 10/1 6pm

```suggestion
<p>If you repeat a particular set of contact details on different pages, show them in the same place on each page.</p>
<p>For example, if you show a ‘contact us’ section at the end of one help page, any other help pages that include the same ‘contact us’ section should have them shown at the end of the page as well.</p>  
```
